tasks/cks/labs/19/SOLUTION.MD

```diff
@@ -1,28 +1,40 @@
-```
-# vim  /var/work/14/Dockerfile
-
+# Read docker [build best practies](https://docs.docker.com/build/building/best-practices/)
 
+# Fix dockerfile
+```
+vim  /var/work/14/Dockerfile
+```
+*It should looks like this one:*
+```
 FROM ubuntu:20.04
-RUN apt-get update
-RUN apt-get -y install curl
+RUN apt-get update && apt-get -y install curl
 RUN groupadd myuser
-RUN useradd  -g myuser  myuser
+RUN useradd -g myuser myuser
 USER myuser
 CMD ["sh", "-c", "while true ; do  id ;  sleep 1  ;done"]
-
 ```
 
+# Build an image
 ```
 podman build . -t cks:14
+```
 
+# Run a container
+```
+podman run -d --rm --name cks-14 cks:14
+```
 
-podman  run -d --name  cks-14   cks:14
-sleep 2
```

Comment:
Sometimes this won't work, as the container may not have time to start. that's why sleep was added.

Reply:
Sleep has been returned